src/style/style.js

```diff
@@ -280,7 +280,7 @@ class Style extends Evented {
         const parameters = {
             zoom: z,
             now: browser.now(),
-            defaultFadeDuration: 300,
```

Comment:
Just to make sure I'm reading this right: 

- The previous behavior was that transitions that didn't specify a duration would default to 300ms
- The new behavior is that transitions that don't specify a duration will inherit the map's (configurable) `fadeDuration`, which also controls collision fading, and which defaults to 300

Reply:
Not quite -- this parameter affects only the cross-fading of `line-dasharray` and `*-pattern` when the zoom crosses an integer threshold. It does not affect property transitions; those are controlled by the top-level `transition` property in the style, overridden by individual `*-transition` properties.